.github/workflows/build-rom.yml

```
@@ -0,0 +1,253 @@
+
+      - name: Cache Rust dependencies
+        uses: Swatinem/rust-cache@v2.9.2
+
+      - name: Install Python
+        uses: actions/setup-python@v7
+        with:
+          python-version: "3.12-dev"
+
+      - name: Setup signing environment
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)